### DISCUSSION TITLE
Disable animations setting

### DISCUSSION THREAD

**Discussion by GerbenClaes** [dailydotdev/daily#1052] [Feature Requests]
I'd like to be able to disable animations with a setting as they make it harder to focus on reading text.

**Comment by idoshamun** [comment]
Hi @GerbenClaes,

Thanks for reaching out! What animations make it harder to read? Do you refer to GIFs or something else?

**Reply by GerbenClaes** [reply]
Thanks for following up on my request!

Yes indeed the GIFs, the ones displayed inside the cards of the articles. 
E.g. when trying to read comments and a card is displayed in the background it grabs the focus and makes it harder to concentrate on the text.
It is also a WCAG guideline, more info can be found here: https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html

**Reply by idoshamun** [reply]
Thanks for the clarification! I'll log it down
